workloads/logging/common.sh

```diff
@@ -14,9 +14,9 @@ log() {
 log "Checking if oc client is installed"
 
 # Check cluster's health
-if [[ ${CERBERUS_URL} ]]; then
-  response=$(curl ${CERBERUS_URL})
-  if [ "$response" != "True" ]; then
+if [[ "${CERBERUS_URL}" ]]; then
```

Comment:
We can remove the entire Cerberus section. It is not used anymore

Reply:
removed